pkgs/client/__tests__/SupabaseBroadcastAdapter.simple.test.ts

```
@@ -0,0 +1,172 @@
+    vi.spyOn(console, 'log').mockImplementation(() => {});
+  });
+
+  afterEach(() => {
+    vi.restoreAllMocks();
+  });
+
+  /**
+   * Helper to create a synchronous scheduler for immediate execution
+   */
+  function createImmediateScheduler() {
+    return (fn: Function) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)